general/administration/configuration.md

```diff
@@ -24,7 +24,7 @@ This is the directory that will hold all Jellyfin data, and is also used as a de
 
 1. Command line option `--datadir`, if specified
 2. Environment variable `JELLYFIN_DATA_DIR`, if specified
-3. `<%APPDATA%>/jellyfin`, if running on Windows
+3. `<%PROGRAMDATA%>/jellyfin`, if running on Windows

```

Comment:
The programdata folder is only used by the Jellyfin Tray app. If you install portable or build the code yourself it uses the programdata folder.

Reply:
Got it.  It seemed like all configs were there in %PROGRAMDATA%.  After breaking my setup with an invalid base url, the network.xml I needed to update was in '%PROGRAMDATA%\Jellyfin\Server\config\network.xml'.